libs/ui/catalog/src/lib/organisation-preview/organisation-preview.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="">
+<div (click)="clickOrganisation()" class="cursor-pointer">
```

Comment:
When an element becomes interactive, giving it a click handler and a cursor is usually not enough. Things to do include:
* use a `<button type="button">` element to make sure the element is accessible by keyboard
* add a `title` attribute to describe the action
* provide visual feedback on hover

Reply:
Thanks for the feedback. I've changed the `div` to an `a` to make it accessible by keyboard and added a shadow on hovering as visual feedback.